doc/ROADMAP.rst.txt

```diff
@@ -250,21 +262,26 @@ for those).
 
 sparse
 ``````
-The sparse matrix formats are getting feature-complete but are slow ...
-reimplement parts in Cython?
-
-    - Small matrices are slower than PySparse, needs fixing
-
-There are a lot of formats.  These should be kept, but
-improvements/optimizations should go into CSR/CSC, which are the preferred
-formats.  LIL may be the exception, it's inherently inefficient.  It could be
-dropped if DOK is extended to support all the operations LIL currently
-provides.  Alternatives are being worked on, see  https://github.com/ev-br/sparr
-and https://github.com/perimosocordiae/sparray.
-
-Ideas for new features:
-
-    - Sparse arrays now act like np.matrix.  We want sparse *arrays*.
+The sparse matrix formats are mostly feature-complete, however the main issue
+is that they act like ``numpy.matrix`` (which will be deprecated in NumPy at
+some point).  What we want is sparse arrays, that act like ``numpy.ndarray``.
+This is being worked on in https://github.com/pydata/sparse, which is quite far
+along.  The tentative plan is:
+
+- Start depending on ``pydata/sparse`` once it's feature-complete enough (it
+  still needs a CSC/CSR equivalent) and okay performance-wise.
+- Add support for ``pydata/sparse`` to ``scipy.sparse.linalg`` (and perhaps to
+  ``scipy.sparse.csgraph`` after that).
+- Indicate in the documentation that for new code users should prefer
+  ``pydata/sparse`` over sparse matrices.
```

Comment:
I'll just say that this is a cool idea and it's nice to see `pydata/sparse` growing into the "main" sparse package for SciPy to depend on!